impl_core/src/main/java/io/opencensus/implcore/trace/export/ExportComponentImpl.java

```diff
@@ -39,17 +39,11 @@ public SpanExporterImpl getSpanExporter() {
   }
 
   @Override
-  // TODO(#914): This method shouldn't be nullable.
-  @SuppressWarnings("nullness")
```

Comment:
You also need remove the `@Nullable` annotation on line 33.

Reply:
Thanks! Done.